androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/ManageDataLauncherActivity.java

```diff
@@ -332,25 +334,65 @@ public static boolean packageSupportsSiteSettings(String packageName,
      *
      * Otherwise returns null if {@link ManageDataLauncherActivity} is not launchable
      * or if shortcuts are not supported by the Android SDK version.
+     *
+     * The shortcut returned does not specify an activity. Thus when the shortcut is added,
+     * the app's main activity will be used by default. This activity needs to define the
+     * MAIN action and LAUNCHER category in order to attach the shortcut.
      */
-    public static ShortcutInfo getSiteSettingsShortcutOrNull(Context context, PackageManager packageManager) {
+    @Nullable
+    public static ShortcutInfo getSiteSettingsShortcutOrNull(Context context,
```

Comment:
Can this be private now? (Or do we want to keep it package-private so we can test it?)

Reply:
I changed it to package private now